testnets/archwaytestnet/chain.json

```diff
@@ -0,0 +1,47 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "archwaytestnet",
+  "chain_id": "constantine-1",
+  "pretty_name": "Archway testnet",
+  "status": "live",
+  "network_type": "testnet",
+  "website": "https://archway.io",
+  "bech32_prefix": "archway",
```

Comment:
Please register this bech32 prefix tp https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
Done: https://github.com/satoshilabs/slips/pull/1510